schemas/Item.yaml

```diff
@@ -16,8 +16,4 @@ properties:
     type: array
     items:
       $ref: Regulator.yaml
-  batch:
```

Comment:
Renaming Each to Item and removing the existing proposed joins is a pretty big change that probably requires a dedicated agenda item on a meeting rather than a PR review. Recommend further discussion.

Reply:
See above re: renaming. We can leave as is. However the change to refer to batch and product by ID vs. including the actual record in the each is an efficiency thing. including the actual batch and product information in an Each record leads to a huge amount of duplication (they would be the majority of the each record if included).

Reply:
We've found its better to refer to the batch and product records, which can be retrieved by ID as needed by the client.

Reply:
I understand your concern here- any reason we can't add a headers or query param to expand those - something like expandFields="batch,whatever" ?